README.adoc

```diff
@@ -328,6 +328,8 @@ To override configuration values using environment variables, use the prefix
 
 For example to override `postgres.password`, set the environment variable `F8_POSTGRES_PASSWORD` to the value of you liking.
 
+NOTE: config.yaml is *not* processed by default. In order to use it, the following environment variable must be set `export F8_CONFIG_FILE_PATH="config.yaml"`
```

Comment:
Yes. Generally, we simply pass the ENV variables that differ from the default values.